src/app/vm-logs/containers/vm-logs-table.container.ts

```diff
@@ -1,19 +1,40 @@
 import { Component } from '@angular/core';
-import { select, Store } from '@ngrx/store';
+import { select, Store, createSelector } from '@ngrx/store';
 import { State } from '../../reducers';
 import * as fromVmLogs from '../redux/vm-logs.reducers';
+import { map } from 'rxjs/operators';
+import * as vmLogsActions from '../redux/vm-logs.actions';
+import * as fromVmLogsAutoUpdate from '../redux/vm-logs-auto-update.reducers';
+
+const selectIsAutoUpdateWithScroll = createSelector(
```

Comment:
why you create selector here?
why don't you use selectors.ts or reducers.ts as usual